src/ixdat/plotters/ec_plotter.py

```diff
@@ -37,18 +44,20 @@ def plot_measurement(
             measurement (Measurement): The measurement to plot, if not the one the
                 plotter was initiated with.
             tspan (iter of float): The timespan (wrt to measurement.tstamp) to plot.
+            axes (list of matplotlib.Axis): Two axes to plot on, if not the default
+                new twinx()'d axes. axes[0] is for `v_name` and axes[1] for `j_name`.
             v_name (string): The name of the ValueSeries to plot on the left y-axis.
                 Defaults to measurement.V_str, which for an ECMeasurement is the name
                 of its most calibrated/correct potential.
             j_name (string): The name of the ValueSeries to plot on the right y-axis.
                 Defaults to measurement.J_str, which for an ECMeasurement is the name
                 of its most normalized/correct current.
-            axes (list of matplotlib.Axis): Two axes to plot on, if not the default
-                new twinx()'d axes. axes[0] is for V_str and axes[1] for J_str.
-            V_str (str): DEPRECIATED. now v_name
-            J_str (str): DEPRECIATED. now j_name
             v_color (str): The color to plot v_name. Defaults to black.
             j_color (str): The color to plot j_name. Defaults to red.
+            V_str (str): DEPRECATED
```

Comment:
Maybe, you should keep the new names here still, so that it is deducible from the API docs only, without running the code.